setup.py

```diff
@@ -64,6 +64,7 @@ def run_tests(self):
     'requests>=2.7.0',
     'ws4py>=0.3,<0.4',
     'zope.sqlalchemy>=0.7.6,<0.8.0',
+    'itsdangerous>=0.24',
```

Comment:
Sorry to be a pain, but these are in alphabetical order, so if we could keep them that way that would be great :)